### DISCUSSION TITLE
求个激活码，大佬

### DISCUSSION THREAD

**Discussion by ytAlbert** [LFTEC/LearningFucker#51] [General]
dJDfdSEQdi03khSjHUSRR2+MdNdzMLYJuAZuxpsP6KUg229Xt7lbrlW1kFbpEdPqfDe
ye2TIjkLSS//pP9o/PSGzf5+PHM1OlsbwCsIK40lP16LpMvo7N5fTsDDmcRancemAxT
S4CSLWvPtmQw+fvWjpI43Gri4h+UFM+JOcrhzFBOhu8L3WfLW9RU85qdcqyCqwNf0yw
POEO19IfmmhxaFCDCKO74W5cHWlD4UbWdfVUt6i7tziPu4NABw6WWSD6caTt4eDNmjJ
da9XukvohSamYi+1naMzc2JjuDvpHyX26vHE2H59fCkFxO232P3ar5D2Wzdo/B6/2CU
CUO3xW791asytcdX8tRmm1RWpM5l4JuRfzPqsERt7ymseoLiYPEzKpE74uurZs0YRzu
D8FQVIK/kN9O/DpgRPV4+UH4I8j8arIeIg8ZFOg7n0rGN/Rz+vGvNRzeq/SKxqm1hG9
N3QpZBjXdjOFhYDLSPWg+isdQ9diOtDp4FBe+Q59DNh2ReSWvbtpQAJ6eM10JbEco1M
x0RQ/ZwJgs+EPV03oA+JBzi9F4zt98DmTuzF0GdEhptMAA5d8LjqTbHUUrHeS0lFPLa
DqYAEv0DTRaD/2QqkGbenfDhJXfAyUZdWUmJ/U4GFoaiPUoZ4lHVx6nlT6nvtj+CCrD
kgQa8aOtJc+cSDL1EfjxdavZFagFv6T7z5dRvCCts9oykBzjIr+RjW5Hi2eg2AQaP9Z
cODz6OEIbL2fmNSsOkZ9wBywu+ilxnZHcws1bwXWh6pFcxabAl9rJjtf5cL9kFaF946
20tr0/ECml1SyQJf42KpDlCvgQig8LGBZPe7wl5E489vV5ukQeT0OtKUeQOKXULzUfW
0qw9Om+ZO0N19iDF2pJkOP6u2dSnZV8D+Q1QNMeo7ZA8nBomCJXqcIfU7qT5Rlj5Y/7
6sWi0/cGYV70AGJ7pWfyxn+e6Y47yLWnzGSLRgPPinNT/8s5orr9mfE4d6CdBzzqgP9
h0l4VBNXUrF3G2LHlUUrEq/Rscu/bYhf//b7CqLuozJMTugS/skAzb0c8gZo5kYf1QW
aFHiELrsxefxgczNC3ip8aw5piAKhUqlkYyAjEo3ud7sEyE+S8iuvYork91r9jbdD2z
SpvlGKdLDot3nLuIJYVKHf+9eNsIOwmF+c2y5JZOR5R+nK+rnESr8SyyfyWYiZZTPVw
/N3CjwKbP7abeHBw0M7EFE6b0iBsLuQG35FJM0SoSn4BIH11dB4hFzN+TFyWEqAqeiP
vR30bf4JMX76OpewUvOex+B/KpG2OS8oCXeAUAI5PO2Ewenx8hbOd98r8U5nUI3nztk
g9GZyeWZF8240ADiyY7J3oi3uw==

**Comment by LFTEC** [comment]
VQAwS2QXWRhtqV9iBnJ1uWTIRWwM+Uzh2cO/ThyKEnbWey6vxk2lou4UsieRWXEWzryX/U5qpqog0/ogqubq9zT2z62hai8dUVeM00BDBXikdFR1I6Tj4uge1ffDhWEZWIPs5KknSO3NimId2DWsEyvXhZQbSH/0JahlZXZQlVckYg8nXk6/P3FEsVUkk/OY4vCJ6SD/NSy7oFiO391EzWuY0AbRk4dIAMJ03jteKS8+RWgPkp5GdjnacJfdPpXhQPGepRMKdIkjwdh05VUzhfY4omijnCC+qOtic8WuI1h9H4NIgogPjBZObwFyY9SONbGxfUScTJ2tVb7uVH0aj1Zz9FyvLv9RH07MtQud5GSCS+Hpf2g721D6aZn7g0pvsYX2/8aYqYRKgsC5IMulwbVd/1Zvq8t1tM9Z67EluxW1KV5xKHr6OECpVTbvZ66NwsGUlKdKY+FlZzSyhV6LK6xrrqhApzAHlz1O91CMZvfverMswIC4ikJr/zQm85PJsTo/R0II64iex1cfIj81UZ5U4aOfzd+Jjowfb1exr1HIJWFcS9obgY8SLw4PdysEesAOSMe93CHSex9c/6hgQImQ45LNnlTHjRU6kdvp+0VbN+KDaBX4ohLmCo+augedrRnD5kEK5xh6Yuw6cqhw42AwlCWwb+CIUorqZDEosAkdS0D6JVfqf0GLegaAI5tpBn+AfsqbkNq3LlldzkxHXFs5WophADO61uRY/5bf7mKDli7FIcdfcw91Sa3Pp1EQbyks6el9WhmZE73lzCw1YTcEsx2f7dj/DVPPnyVNpI21ktvP5dXd6BCOWyILeZamWMsaMzYMcXFSo94wHlGbQA4zdVFmfGNN+aToAgbC+kDCvaF6FSx3OSmLoQYPn0bjjaocHtC13g7jyUt3UeIrMPKxgFdKFPXUYQy+Ym1i6CzknDnrRynDOQj0cjyzzca18p2Hjx6ma8BGCzZM6fxWAPLInzmD/wxWt9yGkogmJFZtApFeYPKFpoUhDGKsG78IbcZBohzuAKSN/DeDoL4ztEq8UK7uG4UgoYlQhqLo01d2c3Aa4RMyiaHXKALIbVHsRzJ7K+AP3XKriKK7xSpKRHDRN5iIEEbt+MkfJkNN4bb1js8VKfbkTH+eYpsrtwOPnSme8zfp6W85lGQ+9Ti8rAqiqDmcmtOBHGr6hRxpZe98UfT6prNDI48/4tPCuUdhPIOgcTQtlqyTsLBfL6g7pz1/5aiqUv7Vb6WglO08H9UQOrgZP0HbD5UawAfhzyrQhenfSpn2t5V4vGiMzbyJQqvD7QdCa6myl/JOuL6AKVj5Nj4/ldar3V1Y1mVoLH5GRIs644WUN/MWvXEWrQ9X2w==
